claims_hosp/tests/test_update_indicator.py

```diff
@@ -35,6 +36,9 @@ class TestClaimsHospIndicatorUpdater:
     weekday = False
     write_se = False
     prefix = "foo"
+    start_date = "02-01-2020"
+    end_date = "06-01-2020"
+    drop_date = "2020-06-12"
```

Comment:
To match the old code and for consistency
```suggestion
    start_date = "02-01-2020"
    end_date = "06-01-2020"
    drop_date = "06-12-2020"
```